ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/zones/customized-resource/provision/cucushift-installer-rehearse-nutanix-ipi-zones-customized-resource-provision-chain.yaml

```diff
@@ -12,5 +12,6 @@ chain:
   - ref: cucushift-installer-check-nutanix-zones
   - ref: cucushift-installer-check-nutanix-customized-resource
   - ref: cucushift-installer-check-nutanix-os-image
+  - ref: cucushift-installer-check-nutanix-vip
```

Comment:
I think this is a common check point, we can add this step to many nutanix install workflows, right?

Reply:
IMO it have not to much meaning to run it in many nutanix workflows, just run it for full regression. What do you think?

Reply:
As what we discussed in OCPQE-17296, the principal is if the test case can be automated and covered in regular ci, then cover it in regular ci, if the test case is only specific for installer team, we can run it in installer dedicated ci.

Per my understanding, the VIP check can be run in most of regular ci, but not supposed to run it in installer dedicated ci. 

If you think the check point might bring some instability issues for regular ci, suggest to make the step simpler, only cover the necessary test point, e.g: check VIPs setting in install-config.yaml is being used in ingress route and api service. From my perspective,  in cucushift-installer-check-nutanix-vip-commands.sh, create new app, scale up app to 3 replicas, access the app url inside cluster and outside cluster are not very related to the purpose of the step, we can directly get existing service route url to check VIP settings.

Reply:
@jianlinliu  Fixed, pls help to review again, thanks.